fetch.bs

```diff
@@ -5106,7 +5106,13 @@ steps:
     <p>When in doubt, return a <a>network error</a>.
 
    <dt><a>HTTP(S) scheme</a>
-   <dd><p>Return the result of running <a>HTTP fetch</a> given <var>fetchParams</var>.
+   <dd>
+    <ol>
+     <li><p>If <var>request</var>'s <a for=request>mode</a> is "<code>navigate</code>", then return
+     the result of running <a>HTTP fetch</a> given <var>fetchParams</var> and true.
+
+     <li><p>Return the result of running <a>HTTP fetch</a> given <var>fetchParams</var>.
```

Comment:
I would keep this a one-liner reading "Return the result of running HTTP fetch with fetchParams and true if request's mode is navigate, and false otherwise."